src/vaadin-button.html

```diff
@@ -147,6 +147,7 @@
           this.$.button.setAttribute('role', 'presentation');
 
           this._addActiveListeners();
+          window.ShadyDOM && window.ShadyDOM.flush();
```

Comment:
Missing a test. Something like what's suggested [here](https://github.com/vaadin/vaadin-button-flow/issues/120#issuecomment-496891538) should work as a test case.

If the fix cannot be tested here in the `vaadin-button`, you need to add a proper description as a comment for this addition. Something that explains why this line is needed.